docs/non-normative/k8s-naming.md

```diff
@@ -0,0 +1,22 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: K8s naming
+--->
+
+# Kubernetes Naming Guidance
+
+When naming resource, attributes, or metrics related to Kubernetes objects, **prioritize consistency with Kubernetes API naming** over certain Opentelemetry's naming conventions, as long as the resulting name is valid per the general naming rules.
+
+**Rationale:**  
+Kubernetes users are accustomed to the single-word resource names used in the Kubernetes API (e.g., `replicaset`, `statefulset`, `daemonset`, `replicationcontroller`, `resourcequota`).  
```

Comment:
You could start with this "rationale" first and then follow with the guidance. Then maybe you don't even need to explicitly call it as a rationale, as it will be just a natural read - k8s stuff are named like this, so in semconv we also recommend doing the same.